source/client/components/CVLanguageManager.ts

```diff
@@ -46,11 +46,12 @@ export default class CVLanguageManager extends Component
 
     protected static readonly ins = {
         enabled: types.Boolean("Language.Enabled", false),
-        language: types.Enum("Interface.Language", ELanguageType, ELanguageType.EN),
+        language: types.Enum("Interface.Language", ELanguageType, ELanguageType[DEFAULT_LANGUAGE]),
     };
 
     protected static readonly outs = {
-        language: types.Enum("Interface.Language", ELanguageType, ELanguageType.EN),
+        /* exception to default language: in absence of any dictionary, this is always EN */
+        language: types.Enum("Interface.Language", ELanguageType, ELanguageType.EN ),
```

Comment:
Why an exception? In the absence of any dictionary, won't DEFAULT_LANGUAGE be EN anyways?

Reply:
In the absence of a dictionary, the "output" language is always `EN` (or rather "untranslated UI strings", in English by convention).

This is not the same as `DEFAULT_LANGUAGE`, as someone might set it to another value.

Because of this I left this comment to not be tempted to replace `ELanguageType.EN` with `DEFAULT_LANGUAGE` as it was done everywhere else because this would be wrong and introduce again places where English is confused with DEFAULT_LANGUAGE, which defeats the idea of a DEFAULT_LANGUAGE global.

Reply:
Fair point, thanks for the explanation.